README.md

```
@@ -31,9 +31,11 @@ and configure these applications.
     - [Build Image](#build-image)
     - [Deployment](#deployment)
       - [Deployment Methods Comparison](#deployment-methods-comparison)
+    - [Image Overrides](#image-overrides)
   - [Cloud Manager (CCM)](#cloud-manager-ccm)
```

markdownlint-cli2 (0.23.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 2

(MD005, list-indent)